core/src/test/scala/unit/kafka/log/LogConfigTest.scala

```diff
@@ -162,6 +162,21 @@ class LogConfigTest {
     assertNull(nullServerDefault)
   }
 
+  @Test
+  def testOverriddenConfigsAsLoggableString(): Unit = {
+    val kafkaProps = TestUtils.createBrokerConfig(nodeId = 0, zkConnect = "")
+    kafkaProps.put("unknown.broker.password.config", "aaaaa")
+    kafkaProps.put(KafkaConfig.SslKeyPasswordProp, "somekeypassword")
```

Comment:
Could we add a line with the same setting as overridden one, but not the same value, so that we can verify the overridden value will be outputted? Ex:
add this line
`kafkaProps.put(LogConfig.MinInSyncReplicasProp, "1")`
then, we already have
` topicOverrides.setProperty(LogConfig.MinInSyncReplicasProp, "2")`

So, we can assert the log is output `min.insync.replicas=2`

Reply:
Yes, we can. The goal of this test is not to verify that, but no harm in covering it too.